<!DOCTYPE html>
<html lang="en-US" dir="ltr">
<!--********************************************-->
<!--*       Generated from PreTeXt source      *-->
<!--*                                          *-->
<!--*         https://pretextbook.org          *-->
<!--*                                          *-->
<!--********************************************-->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="robots" content="noindex, nofollow">
</head>
<body class="ignore-math">
<article class="technology computation-like"><h4 class="heading">
<span class="type">Technology</span><span class="space"> </span><span class="codenumber">A.6.14</span><span class="period">.</span><span class="space"> </span><span class="title">Python <code class="code-inline tex2jax_ignore">continue</code> statements.</span>
</h4> <div class="para">When <code class="code-inline tex2jax_ignore">continue</code> is encountered during the execution of a loop, the current step of the loop immediately completes and the loop condition is reevaluated, moving to the next object in a <code class="code-inline tex2jax_ignore">for</code> loop or retesting the conditional of a <code class="code-inline tex2jax_ignore">while</code> loop.</div> <div class="para">This is particularly useful in cases where the code must handle cases for the loop which do nothing, or are exempt from the behavior encoded in the block structure of the loop, but that do not halt the execution of the loop.</div></article><span class="incontext"><a class="internal" href="sec-intro-errors.html#tech-continue">in-context</a></span>
</body>
</html>
